src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -59,14 +59,15 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
     TakeLock
     if [ ! -d IL-CG2 ]%3B then
       mkdir IL-CG2
+      cp *.dll IL-CG2/
+      rm IL-CG2/composite-r2r.dll 2>/dev/null
+      rm IL-CG2/Coreclr.TestWrapper.dll 2>/dev/null
+      rm IL-CG2/*.XUnitWrapper.dll 2>/dev/null
+
       if [ ! -z ${CompositeBuildMode+x} ]%3B then
-        cp $(AssemblyName).dll IL-CG2/
+        # FIXME: Why is this cp here? Windows doesn't do it.
```

Comment:
These are native shared shim libraries that Windows don't have. But I don't know why we need to copy them there at the moment.